internal/clients/unisat/unisat.go

```diff
@@ -0,0 +1,100 @@
+package unisat
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	baseclient "github.com/babylonchain/staking-api-service/internal/clients/base"
+	"github.com/babylonchain/staking-api-service/internal/config"
+	"github.com/babylonchain/staking-api-service/internal/types"
+)
+
+type UnisatInscriptions struct {
+	InscriptionId     string `json:"inscriptionId"`
+	InscriptionNumber int    `json:"inscriptionNumber"`
+	IsBRC20           bool   `json:"isBRC20"`
+	Moved             bool   `json:"moved"`
+	Offset            int    `json:"offset"`
+}
+
+type UnisatUtxos struct {
```

Comment:
```suggestion
type UnisatUTXO struct {
```
Refers to a single UTXO. Also capitalizing to go with the convention in this repo